src/main/java/org/craftercms/studio/api/v2/dal/publish/PublishItem.java

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright (C) 2007-2024 Crafter Software Corporation. All Rights Reserved.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as published by
+ * the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.craftercms.studio.api.v2.dal.publish;
+
+/**
+ * Represents an item to be published.
+ */
+public class PublishItem {
+    protected long id;
+    protected long packageId;
+    protected String path;
+    protected String liveOldPath;
+    protected String stagingOldPath;
+    protected Action action;
+    protected boolean userRequested;
+    protected State state;
+    protected String error;
```

Comment:
Is this where we would add the new fields?

Reply:
The item metadata/cached fields like label, systemType, etc. Yes.